utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/JsonSinkValueParameter.scala

```diff
@@ -24,7 +24,7 @@ object JsonSinkValueParameter {
   private def toSinkValueParameter(schema: Schema, paramName: Option[String], defaultParamName: String, defaultValue: Option[Expression], isRequired: Option[Boolean])
                                   (implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SinkValueParameter] = {
     schema match {
-      case objectSchema: ObjectSchema =>
+      case objectSchema: ObjectSchema if !(objectSchema.getPropertySchemas.isEmpty && objectSchema.permitsAdditionalProperties()) =>
```

Comment:
can we extract `objectSchema.getPropertySchemas.isEmpty && objectSchema.permitsAdditionalProperties()` to a method?
my point is to have descriptive name, cause this expressions means, that schema represent Map, right?

Reply:
actually we have it already. `JsonSchemaOutputValidator` provides with implicit class with this method

Reply:
:+1: I moved `representsMap` to `JsonSchemaImplicits`